src/plugins/api/apitypes.h

```diff
@@ -408,8 +408,8 @@ enum class BeamMode {
     MID = int(mu::engraving::BeamMode::MID),
     END = int(mu::engraving::BeamMode::END),
     NONE = int(mu::engraving::BeamMode::NONE),
+    BEGIN16 = int(mu::engraving::BeamMode::BEGIN16),
     BEGIN32 = int(mu::engraving::BeamMode::BEGIN32),
-    BEGIN64 = int(mu::engraving::BeamMode::BEGIN64),
```

Comment:
You should keep this value (and add a comment that it's deprecated) since it's used for the API

Reply:
done!